### PR TITLE
More configuration from environment, use TOKEN_AUTH_* for authentication

### PR DESCRIPTION
**Pull request by vikoivun**

### CODE REVIEW

File: respa/settings.py [hunk 1]
```diff
@@ -17,8 +17,9 @@
     ALLOWED_HOSTS=(list, []),
     ADMINS=(list, []),
     DATABASE_URL=(str, 'postgis:///respa'),
-    JWT_SECRET_KEY=(str, ''),
-    JWT_AUDIENCE=(str, ''),
+    SECURE_PROXY_SSL_HEADER=(tuple, None)
```
Comment (tuomas777) [inline respa/settings.py]: Missing a comma at the end.

**Comment by jussiarpalahti** [comment]
@vikoivun So, replacing variables are same as replaced variables, only names are changed? No change in expected values?

**Comment by vikoivun** [comment]
> @vikoivun So, replacing variables are same as replaced variables, only names are changed? No change in expected values?

Yep. It just tries to show in the name, that is has to do with tokens and authentication.

**Comment by vikoivun** [comment]
The sentry logging thing is a weird one. At one point at least we had a case where anymail had just silently eaten any errors with sending mail. But recently when mailgun reached the free message limit, anymail instead raised the exception causing the whole reservation to fail.

**Comment by jussiarpalahti** [comment]
What sentry logging thing? I see there's both raven and sentry error loggers in config now. Are both really necessary?

**Comment by vikoivun** [comment]
> What sentry logging thing? I see there's both raven and sentry error loggers in config now. Are both really necessary?

As I said, at some point I was certain that anymail logged errors and I wanted to capture them through sentry. But now I looked through anymail code and there are no loggers at all. I'll remove the logging.

**Comment by jussiarpalahti** [comment]
Well, yes, but there is configs for raven (the old sentry) and sentry.errors which I take is for new SDK. Can both truly work with either Python package?

**Pull request merged**